source/Irrlicht/CMakeLists.txt

```diff
@@ -44,19 +38,21 @@ if(OGLES1_ENABLED)
 	find_library(EGL_LIBRARY NAMES EGL REQUIRED)
 
 	message(STATUS "Found OpenGLES: ${OPENGLES_LIBRARY}")
-	link_libraries(${OPENGLES_LIBRARY} ${EGL_LIBRARY})
+	link_libraries("${OPENGLES_LIBRARY}" "${EGL_LIBRARY}")
 endif()
 check_symbol_exists(_IRR_COMPILE_WITH_OGLES2_ "IrrCompileConfig.h" OGLES2_ENABLED)
 if(OGLES2_ENABLED)
 	find_package(OpenGLES2 REQUIRED)
-	include_directories(${OPENGLES2_INCLUDE_DIR} ${EGL_INCLUDE_DIR})
+
+	include_directories("${OPENGLES2_INCLUDE_DIR}", "${EGL_INCLUDE_DIR}")
 	link_libraries(${OPENGLES2_LIBRARIES})
 endif()
 check_symbol_exists(_IRR_COMPILE_WITH_OPENGL_ "IrrCompileConfig.h" OGL_ENABLED)
 if(OGL_ENABLED)
 	set(OpenGL_GL_PREFERENCE "LEGACY")
 	find_package(OpenGL REQUIRED)
-	include_directories(${OPENGL_INCLUDE_DIR})
+
+	include_directories("${OPENGL_INCLUDE_DIR}")
 	link_libraries(${OPENGL_LIBRARIES})
```

Comment:
Shouldn't this be removed here since it exists in the target definition?

Reply:
could it be, that the this `include_directories()` call is primarily used because of the many object-libraries? To not have to `target_link_libraries()` each object-library-target?

Reply:
Precisely. I have a comment about that in one of the commits.

Reply:
This works on Linux without including it globally but (of course...) mingw will club me from behind if I try to get away with that.